salt/states/file.py

```diff
@@ -6683,6 +6683,10 @@ def prepend(
         appending content, one space or multiple tabs are the same for salt.
         Set this option to ``False`` if you want to change this behavior.
 
+    header
+        Forces the text to be prepended if it exists in the file but not at
+        the beginning.
```

Comment:
> "but not at the beginning.

I tested the header option and it does add it to the beginning. Essentially the way i'm interpreting header to act like is it will ensure the text you set will always be at the beginning of the file, verbatim. 

Also, if the text does not exist it will still prepend it at the beginning of the file. 

I would suggest:

"Forces the text to be prepended at the beginning of the file verbatim."

Reply:
"but not at the beginning" applies to "exists in the file".

If the text already exists in the file, but not at the beginning, then it prepends a duplicate.

Reply:
How about:  "Forces the text to be prepended. If it exists in the file but not at the beginning, then it prepends a duplicate"?